android/build.gradle

```diff
@@ -32,6 +32,8 @@ android {
 dependencies {
     api project(':core')
 
+    api 'org.slf4j:slf4j-api:2.0.7'
```

Comment:
I think this can be an `implementation` dependency? I didn't see it in any public members or method signatures, but I could have missed some of course.